source/github-helpers/selectors.ts

```diff
@@ -44,6 +44,11 @@ export const prCommitStatusIcon_ = [
 	'https://github.com/refined-github/sandbox/pull/10',
 ];
 
+export const prOpenOrDraftRow = '.js-issue-row:has(.octicon-git-pull-request.color-fg-open,.octicon-git-pull-request-draft)';
```

Comment:
- Added color check like this : `.octicon-git-pull-request.color-fg-open` because of below case.
![image](https://github.com/refined-github/refined-github/assets/50487467/b23bb866-4cae-433f-91bf-f76c8b9c4da6)